src/index.ml

```diff
@@ -4,6 +4,57 @@ module Private = struct
   module Search = Search
 end
 
+module Stats = struct
```

Comment:
Did you consider defining this in `stats.ml`?

Reply:
I tried :) but I would like to call it from two files in different folders (`index.ml` and `index_unix.ml`) and avoid making it a library, and I didn't managed to do it otherwise.

Reply:
as there is in an `index.mli` the `Stats` module will be hidden by default. You need to add `module Stats = Stats` to export it in the public interface.

Reply:
Thanks, it worked !